src/components/Form.js

```diff
@@ -0,0 +1,39 @@
+import {nanoid} from 'nanoid';
+import {useState} from 'react';
+
+import StyledForm from '../style/StyledForm';
+
+import {Workouts} from './db';
```

Comment:
Personal preference: You can do it this way. But for me the `db.js` isn't a component. It's just a set of data. Therefore you could probably create a JSON file and put it into `src`.